test/e2e/import_test.go

```diff
@@ -62,11 +62,9 @@ var _ = Describe("Podman import", func() {
 		export.WaitWithDefaultTimeout()
 		Expect(export.ExitCode()).To(Equal(0))
 
-		importImage := podmanTest.PodmanNoCache([]string{"import", outfile})
+		importImage := podmanTest.Podman([]string{"import", outfile})
 		importImage.WaitWithDefaultTimeout()
 		Expect(importImage.ExitCode()).To(Equal(0))
-
-		Expect(podmanTest.ImageExistsInMainStore(importImage.OutputToString())).To(BeTrue())
```

Comment:
podman images shows readwrite and readonly images, we could get these from this command.